bluepyemodel/efeatures_extraction/targets_configuration.py

```diff
@@ -101,7 +103,66 @@ def __init__(
+            additional_fitness_efeatures (list of dicts): efeatures to add to
+                the output of the extraction, i.e. to the FitnessCalculatorConfiguration (FCC).
+                These efeatures will not be extracted from the targets,
+                but will be used during optimisation and / or validation.
+                They should have the same format as the efeatures of the FCC, e.g.
+
+                .. code-block::
+
+                    [
+                        {
+                            "efel_feature_name": "Spikecount",
+                            "protocol_name": "IDrest_130",
+                            "recording_name": "soma.v",
+                            "mean": 6.026,
+                            "original_std": 4.016,
+                            "efeature_name": "Spikecount",
+                            "efel_settings": {
+                                "strict_stiminterval": true,
+                                "Threshold": -30.0,
+                                "interp_step": 0.025,
+                                "stim_start": 700.0,
+                                "stim_end": 2700.0
+                            }
+                        }
+                    ]
+            additional_fitness_protocols (list of dicts): protocols to add to
+                the output of the extraction, i.e. to the FitnessCalculatorConfiguration (FCC).
+                These protocols will not be used during extraction,
+                but will be used during optimisation and / or validation.
+                They should have the same format as the protocols of the FCC, e.g.
+
+                .. code-block::
+
+                    [
+                        {
+                            "name": "IDrest_130",
+                            "stimuli": [
+                                {
+                                "delay": 663.3473451327434,
+                                "amp": 0.3491827776582547,
+                                "thresh_perc": 130.56103157894464,
+                                "duration": 2053.050884955752,
+                                "totduration": 3000.0,
+                                "holding_current": -0.13307330411636328
+                                }
+                            ],
+                            "recordings_from_config": [
+                                {
+                                "type": "CompRecording",
+                                "name": "IDrest_130.soma.v",
+                                "location": "soma",
+                                "variable": "v"
+                                }
+                            ],
+                            "validation": false,
+                            "protocol_type": "ThresholdBasedProtocol",
+                            "stochasticity": false
+                        }
+                    ]
+             protocols_mapping (dict, optional): maps original protocol
```

Comment:
Smaller version of these docstrings can be in the pipelinesettings as well.

Reply:
But these variable are not in pipeline settings, I don't think we should add dosctrings where they don't belong

Reply:
Ok. I thought you also intended to add a pipeline setting for these two additional dictionaries.

Reply:
No, no. Those are set by the user in the TargetsConfiguration.
At first I hesitated at whether I should put those in the PipelineSettings or in the TargetsConfiguration, and I decided that it would make more sense to put them in the TargetsConfiguration.
Sorry if I mislead you